core/postsolve.gms

```diff
@@ -23,7 +23,7 @@ $endif.neg
 );
 
 if(cm_iterative_target_adj eq 4,
-*JeS* Update tax levels/ multigasbudget values to reach the CO2 FF&I budget (s_actualbudgetco2 runs from 2020-2100)
+*JeS* Update tax levels / cumulative GHG emission level (CO2+CH4+N2O, in CO2eq units) to reach the CO2 FF&I budget (s_actualbudgetco2 runs from 2020-2100, not peak budget)
```

Comment:
We update either the emission tax or the emission budget, which can be a CO2 only or a multigas budget, depending on cm_multigasscen. Therefore, I would not specify here, but simply say "Update emission tax levels / budgets to reach the CO2 FF&I budget". Same for the following lines

Reply:
thanks! fixed in the new commit

I use cumulative emission level to mean what the model produces in-iteration, and reserve the word "budget" for the parameter which is the cumulative emission target/emission budget